OptionSelector/CQTableViewController.m

```diff
@@ -0,0 +1,153 @@
+//
+//  CQTableViewController.m
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import "CQTableViewController.h"
+#import "CQCategory.h"
+#import "CQDetailTableViewController.h"
+
+
+
+@interface CQTableViewController ()
+@property (nonatomic) NSMutableArray *categories;
+
+@end
+
+@implementation CQTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.categories = [[NSMutableArray alloc]init];
+    CQCategory *saurischia = [[CQCategory alloc]init];
+    CQCategory *ornithischia = [[CQCategory alloc]init];
+    CQCategory *militaryPlanes = [[CQCategory alloc]init];
+    saurischia.name = @"Saurischian Dinosaurs";
+    ornithischia.name = @"Ornithischian Dinosaurs";
+    militaryPlanes.name = @"Military Aircraft";
+    saurischia.options = @[
+                           @"Brachiosaurus",
+                           @"Apatosaurus",
+                           @"Camarasaurus",
+                           @"Argentinosaurus",
+                           @"Tarbosaurus",
+                           @"Tyrannosaurus",
+                           @"Crylophosaurus",
+                           @"Deinonychus"
+                           ];
+    ornithischia.options = @[
+                             @"Lambeosaurus",
+                             @"Corythosaurus",
+                             @"Styracosaurus",
+                             @"Chasmosaurus",
+                             @"Iguandodon",
+                             @"Psittacosaurus",
+                             @"Ankylosaurus",
+                             @"Nodosaurus"
+                             ];
+    militaryPlanes.options = @[
+                               @"AV-8B Harrier II",
+                               @"A-10 Thunderbolt II",
+                               @"F-15 Eagle",
+                               @"F-22 Raptor",
+                               @"F-35 Lightning II",
+                               @"Sukoi Su-37",
+                               @"MiG-29",
+                               @"F/A-18E Super Hornet"
+                               ];
+    [self.categories addObject:saurischia];
+    [self.categories addObject:ornithischia];
+    [self.categories addObject:militaryPlanes];
+    
+   
+}
+-(void)viewWillAppear:(BOOL)animated {
+    [self.tableView reloadData];
+}
+
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath  {
```

Comment:
Nicely done. Another way to do this would be to set up the segue in your storyboard file in Interface Builder and then use the `prepareForSegue:` method to pass the selected category to your detail controller.